src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -203,6 +202,114 @@ describe("Editor Page Component", () => {
         expect(saveProjectSpy).toBeCalledWith(expect.objectContaining(partialProject));
     });
 
+    it("When an image is updated the asset metadata is updated", async () => {
+        const testProject = MockFactory.createTestProject("TestProject");
+        const store = createStore(testProject, true);
+        const props = MockFactory.editorPageProps(testProject.id);
+        const wrapper = createComponent(store, props);
+        const imageAsset = testAssets[0];
+
+        await MockFactory.flushUi();
+        wrapper.update();
+
+        const editedImageAsset: IAssetMetadata = {
+            asset: imageAsset,
+            regions: [MockFactory.createMockRegion()],
+        };
+
+        const saveMock = assetServiceMock.prototype.save as jest.Mock;
+        saveMock.mockClear();
+
+        wrapper.find(Canvas).props().onAssetMetadataChanged(editedImageAsset);
+        await MockFactory.flushUi();
+
+        const editorPage = wrapper.find(EditorPage).childAt(0) as ReactWrapper<IEditorPageProps, IEditorPageState>;
+
+        // Image asset is updated
+        expect(assetServiceMock.prototype.save).toBeCalledWith({
+            asset: {
+                ...imageAsset,
+                state: AssetState.Tagged,
+            },
+            regions: editedImageAsset.regions,
+        });
+
+        const matchingParentAsset = editorPage.state().assets.find((asset) => asset.id === imageAsset.id);
+        expect(matchingParentAsset.state).toEqual(AssetState.Tagged);
+    });
+
+    describe("Editing Video Assets", () => {
+        let wrapper: ReactWrapper;
+        let videoAsset: IAsset;
+        let videoFrames: IAsset[];
+
+        beforeEach(async () => {
+            const testProject = MockFactory.createTestProject("TestProject");
+            videoAsset = MockFactory.createVideoTestAsset("TestVideo");
+            videoFrames = MockFactory.createChildVideoAssets(videoAsset);
+            const projectAssets = [videoAsset].concat(videoFrames);
+            testProject.assets = _.keyBy(projectAssets, (asset) => asset.id);
+
+            const store = createStore(testProject, true);
+            const props = MockFactory.editorPageProps(testProject.id);
+
+            wrapper = createComponent(store, props);
+
+            await MockFactory.flushUi();
+            wrapper.update();
+        });
+
+        it("Child assets are not included within editor page state", () => {
```

Comment:
does child here refer to the assets inside a container asset (so image wouldn't count)?

Reply:
Child meaning if the asset has a parent, ex) a VideoFrame